ipwb/util.py

```diff
@@ -39,16 +43,26 @@
 dtPattern = re.compile(r"^(\d{4})(\d{2})?(\d{2})?(\d{2})?(\d{2})?(\d{2})?$")
 
 
-def isDaemonAlive(hostAndPort="{0}:{1}".format(IPFSAPI_HOST, IPFSAPI_PORT)):
+def createIPFSClient():
+    try:
+        return ipfsapi.Client(IPFSAPI_MUTLIADDRESS)
+    except ipfsapi.exceptions.AddressError:
```

Comment:
Is this the right exception here? I have seen many different exception being raised based on the argument.

Reply:
That's the one I encountered when entering bad multiaddresses.

Reply:
We might make it more generic here unless we are confident about the exception chain.

Reply:
Added a second in bdecf23. We don't want to reinvent the ipfsapi test module, so the two ought to be sufficient for now.